.github/workflows/format-check.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
+    dprint-check:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions/setup-node@v3
+        - run: dirty=$(npx dprint check | grep "Found * not formatted file*.") >> "$GITHUB_STEP_SUMMARY"
```

Comment:
The status code is the thing to check here; I would just make this `dprint check` and that's it.

Reply:
Note that the status thing was changed, but IIRC exit codes inside `$()` get swallowed?